libraries/adsk/adsklib/adsklib_defs.mtlx

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0"?>
+<!--
+ Copyright (c) 2021 Autodesk, Inc.
+ All rights reserved.
+ 
+ These coded instructions, statements, and computer programs contain
+ unpublished proprietary information written by Autodesk, Inc., and are
+ protected by Federal copyright law. They may not be disclosed to third
+ parties or copied or duplicated in any form, in whole or in part, without
+ the prior written consent of Autodesk, Inc. 
+ 
+ DESCRIPTION: Node Defintions for Autodesk Advanced Materials
+ VERSION: 1.0.1
+ SHA: $Id$
```

Comment:
If you want to keep this copyright when saving then you'll need to move it to after the <materialx> declaration. We can turn on saving/restoring comments by default for out builds.